packages/clay-color-picker/src/Custom.tsx

```diff
@@ -106,14 +108,15 @@ interface IProps {
  */
 const ClayColorPickerCustom: React.FunctionComponent<IProps> = ({
 	colors,
+	disablePalette,
 	label,
 	onChange,
 	onColorsChange,
 	spritemap,
 }) => {
 	const inputRef = React.useRef(null);
 	const [activeSplotchIndex, setActiveSplotchIndex] = React.useState(0);
-	const [editorActive, setEditorActive] = React.useState(false);
+	const [editorActive, setEditorActive] = React.useState(disablePalette);
```

Comment:
Maybe, you don't even need to use the `disablePalette` as an initial value for the state.

Reply:
We need to set the initial value since the custom interface is only showed after toggling the icon. If we disable the palette, we want to show the custom interface be default